Lecture2/task1/Vegetable.java

```diff
@@ -0,0 +1,12 @@
+abstract public class Vegetable {
+    protected int calories;
+
+    public int getCalories() {
+        return calories;
+    }
+
```

Comment:
add constructor
public Vegetable(int grams, int calories) {
        this.grams = grams;
        this.calories = calories;
    }

Reply:
added